Feature/branding

PensoGlide opened the pull request:
**Description**:
This PR enables the ability to change the appearance of Guardian in a more customizable way. The inputs allowed to be changed are the following:
- Header colour;
- Primary colour;
- Company name;
- Company Logo;
- Login banner;
- Website favicon.

On the frontend, a branding page has been created exclusively for this purpose, that can be accessed only by Standard Registries clicking on the user icon as seen in the following images.
<img width="385" alt="Screenshot 2023-05-16 at 09 11 48" src="https://github.com/hashgraph/guardian/assets/91878554/a96daecd-1da9-4b83-affa-4f29e85d2ddc">
<img width="1420" alt="Screenshot 2023-05-23 at 14 45 48" src="https://github.com/hashgraph/guardian/assets/91878554/8867f156-8075-4a28-b2c1-125853079c04">


In this page, besides saving the configuration, it is possible to preview the changes applied, bearing in mind that these would disappear once the page has been reloaded:
<img width="1420" alt="Screenshot 2023-05-23 at 14 46 30" src="https://github.com/hashgraph/guardian/assets/91878554/6e35df84-5036-451a-a4c3-8d781b95d485">
<img width="1420" alt="Screenshot 2023-05-23 at 14 46 42" src="https://github.com/hashgraph/guardian/assets/91878554/2a17853f-32ce-4ac8-b721-46741ea9c710">

Upon saving, the changes should be persistent throughout the use of Guardian:
<img width="236" alt="Screenshot 2023-05-16 at 09 13 33" src="https://github.com/hashgraph/guardian/assets/91878554/6febdd58-f160-459b-ab8e-f15965b9a822">
<img width="1278" alt="Screenshot 2023-05-16 at 10 17 30" src="https://github.com/hashgraph/guardian/assets/91878554/c44043c0-b030-4348-95c7-b3b83011a572">

For the persistence of the data, the `/branding` endpoint was created with both POST and GET requests in order to push and fetch the changes from the Mongo database. It is worth noting that the `/branding` POST method is restricted to Standard Registries.

In order to reset the changes made to the branding features that came originally with Guardian, simply click the "Reset" button.
<img width="1420" alt="Screenshot 2023-05-23 at 14 45 48" src="https://github.com/hashgraph/guardian/assets/91878554/34da69b7-b93e-417b-ae45-3e06ff48a25b">


**Related issue(s)**:

Fixes #1796 


**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

anvabr commented:
Have Varsha tested this @PensoGlide? We tried to build/assemble/run this and it does not work.

PensoGlide commented:
Yes, it was tested before I changed this PR from draft to a definite one. Afterwards I just merged with the 'develop' branch and fixed the conflicts on the Swagger page and on `api-gateway/src/app.ts` (which was a single line of code). I am looking into it
